File: Duran_TuranMert.py
n = 10
print ('--------*-Pretest-*--------')
while (n > 0):
   print (n)
   n = n-1
   if n == 9:
      pass
      print ("Pass Block at 9!")
   if n == 7:
      n = n - 1
      print ("Continue at 7!")
      continue
   if n == 5:
      print ("Break at 5!")
      break

print ('--------*-There is no posttest-*--------')

File: Duran_TuranMert.rb
$n = 10
  puts("--------*-Pretest-*--------")

while $n > 0  do
  if $n == 9 then
    puts("--------*-Next at 9-*--------")
    $n = $n-1
    next
  end
  if $n == 5 then
    puts("--------*-Break at 5-*--------")
    break
  end

  puts($n)
  $n = $n - 1
end
puts("--------*-Pretest---Until-do*--------")
$i = 10
until $i < 0  do
  if $i == 9 then
    puts("--------*-Next at 9-*--------")
    $i = $i-1
    next
  end

  if $i == 5 then
    puts("--------*-Break at 5-*--------")
    break
  end
puts($i)
$i = $i - 1
end

  puts("")
  puts("--------*-Posttest---Begin - while*--------")
$num = 10
begin
  if $num == 9 then
    puts("--------*-Next at 9-*--------")
    $num = $num-1
    next
  end
  if $num == 5 then
    puts("--------*-Break at 5-*--------")
    break
  end
   puts($num)
   $num -= 1
end while $num > 0